src/MongoDB.Bson/Serialization/Serializers/GuidSerializer.cs

```diff
@@ -56,9 +53,25 @@ public GuidSerializer(BsonType representation)
             }
 
             _representation = representation;
+            _guidRepresentation = GuidRepresentation.Unspecified;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GuidSerializer"/> class.
+        /// </summary>
+        /// <param name="guidRepresentation">The Guid representation.</param>
+        public GuidSerializer(GuidRepresentation guidRepresentation)
+        {
+            _representation = BsonType.Binary;
+            _guidRepresentation = guidRepresentation;
         }
 
         // public properties
+        /// <summary>
+        /// Gets the Guid representation.
+        /// </summary>
+        public GuidRepresentation GuidRepresentation => _guidRepresentation;
```

Comment:
This new property supports moving configuring Guid representations from the `BsonDefaults.GuidRepresentation` global or the `MongoClientSettings.GuidRepresentation` setting to the serializer.

This allows explicit configuration of the Guid representation of an individual serializer, which in turns allows for different POCO Guid fields to be configured with a different Guid representation if necessary.

Reply:
Didn't realize there will be a `MongoClientSettings.GuidRepresentation`.  Is that new?